HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -27,11 +29,14 @@ HLTBTagHarvestingAnalyzer::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGet
 	{
 		dqmFolder_hist = Form("HLT/BTag/Discriminator/%s",hltPathNames_[ind].c_str());
 		std::string effDir = Form("HLT/BTag/Discriminator/%s/efficiency",hltPathNames_[ind].c_str());
+		std::string relationsDir = Form("HLT/BTag/Discriminator/%s/HEP17_HEM17",hltPathNames_[ind].c_str());
 		ibooker.setCurrentFolder(effDir);
 		TH1 *den =NULL;
 		TH1 *num =NULL; 
 		std::map<TString,TH1F> effics;
 		std::map<TString,bool> efficsOK;
+		std::map<TString,std::map<std::string,TH1F> > efficsmod;
```

Comment:
Could you please comment more, why it should be std::string?
The first key is of a TString type:
https://github.com/natalia-korneeva/cmssw/blob/HEP17-btaghlt-validation/HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc#L60
https://github.com/natalia-korneeva/cmssw/blob/HEP17-btaghlt-validation/HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc#L44